_docs/tasks/egress.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Accessing a Service with Egress
+headline: Accessing a Service with Egress
+sidenav: doc-side-tasks-nav.html
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+
+This task describes how to configure Istio to expose an external service to a Kubernetes cluster. You'll learn how 
+to create an Egress proxy, define an external service and make requests to the service from within the cluster.
+
+
+## Doing ...
+
+## Before you begin
+
+This task assumes you have deployed Istio on Kubernetes.  If you have not done so, please first complete
+the [Installation Steps]({{site.bareurl}}/docs/tasks/istio-installation.html).
+
+This task also assumes you have a publicly accessible service to call from within the cluster 
+(or [httpbin.org](http://httpbin.org) can be used as an example). 
+
+### Setup the environment
+
+Create the Istio Egress proxy.
```

Comment:
@rshriram @kyessenov I was wondering about this, should it be part of the `istio.yaml` in istio/istio repo?  Then it would be deployed as part of the istio-installation here: https://github.com/istio/istio.github.io/blob/master/_docs/tasks/istio-installation.md

Or should the egress proxy not be created by default like ingress?